test/services/dumpers/agent-nodejs/agent-nodejs.test.ts

```diff
@@ -0,0 +1,478 @@
+import Context from '@forestadmin/context';
+import appRoot from 'app-root-path';
+import fs from 'fs';
+import os from 'os';
```

Comment:
To assert a correct behaviour on os, shouldn't we mock the isLinuxOs utils ?

Reply:
The less we mock on integration tests, the better. We mock the `os.platform()` call.

Reply:
🔴